site-wireframe.md

```diff
@@ -0,0 +1,60 @@
+# Home Page
+
+## Hero
+
+Hero can either include a static image of what our app will look like, displaying some notion of an audio conversation in another language. 
+
+## Features
+
+A list of features that make us different 
+
```

Comment:
@spencer-hong  any other features we should advertise here?

Reply:
- Open-world interactions towards an defined learning objective
- Instant interaction to customize speed, detail, and ___